src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java

```diff
@@ -0,0 +1,71 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.widget;
+
+import org.apache.commons.lang.NotImplementedException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.commons.lang.NotImplementedException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)